src/main/java/core/basesyntax/service/amount/BalanceActivityHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.amount;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.dao.FruitTransactionDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceActivityHandler implements ActivityHandler {
+    private final FruitTransactionDao fruitTransactionDao = new FruitTransactionDaoImpl();
```

Comment:
Here and in other handlers, it is better to initialise class members with constructor